client/lib/titan/get-titan-urls.js

```diff
@@ -30,14 +31,14 @@ function getTitanUrl( email, app = TITAN_APPS.EMAIL, clearPreviousSessions = fal
 	return titanAppUrl.href;
 }
 
-export function getTitanCalendarlUrl( email ) {
-	return getTitanUrl( email, TITAN_APPS.CALENDAR );
+export function getTitanCalendarlUrl( domain, email ) {
```

Comment:
Could we take advantage of this pull request to fix that typo?

```suggestion
export function getTitanCalendarUrl( domain, email ) {
```

Reply:
Yes!